youtube/generic.md

```diff
@@ -1,37 +1,47 @@
-
----------------------------------------------------------------------------------------------------
-
-A BIG thank you to my GitHub Sponsors:
-***
-Stephen Mount http://github.com/stemount
-Nicholas Carrigan https://github.com/nhcarrigan
-***
-
-- Santosh Yadav https://github.com/santoshyadavdev
-- Andrew Cunliffe https://github.com/andrew-cunliffe
-- Alexander Stoichkov https://github.com/SashoStoichkov
-- Allan Regush https://github.com/AllanRegush
-
----------------------------------------------------------------------------------------------------
-
-Don’t forget to support my channel by subscribing below, it’s free, and also share with your friends. Subscribe now!!
-https://www.youtube.com/eddiejaoude?sub_confirmation=1
-
-Follow on other socials for behind the scenes footage, join discord to continue the conversation...
-
-http://eddiejaoude.io 
-https://github.com/eddiejaoude
-http://youtube.com/eddiejaoude
-http://twitter.com/eddiejaoude
-http://instagram.com/eddiejaoude
-http://linkedin.com/in/eddiejaoude
-https://discord.com/invite/jZQs6Wu
-http://github.com/EddieHubCommunity
+## YouTube Channel
+Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
+**Make sure to tell your friends about it, so that they also learn something new!**
+
+## YouTube Shorts
+I have another YouTube channel for short clips, please **take a moment subscribing it: https://youtube.com/eddiejaoudetv?sub_confirmation=1**
+
+## Join Our Community
+Join our inclusive open source community EddieHub now for **free** at https://www.eddiehub.org
+ 
+---
+
+Make sure to checkout my website and sign up to our community **newsletter**: https://www.eddiejaoude.io for my live stream + video schedule and community calls
+
+Join Our **Community GitHub Organization**:
+ 
+- Go to https://github.com/EddieHubCommunity/support/issues/new/choose
+- Choose The Issue Type Of **Invite Me To The Organization**
+- Choose The Issue Type Of **Invite Me To The Organization**
+- Open The Issue
+- You get an invitation to the community by our GitHub Action 🎉
+
+---
+
+### GitHub Sponsors
+- Stephen Mount: https://github.com/stemount
+- Nicholas Carrigan: https://github.com/nhcarrigan
+- Santosh Yadav: https://github.com/santoshyadavdev
+- Andrew Cunliffe: https://github.com/andrew-cunliffe
+- Alexander Stoichkov: https://github.com/SashoStoichkov
+- Allan Regush: https://github.com/AllanRegush
+ 
+---
+ 
+## Social Media
+Follow on socials for behind the scenes footage and more.
+
+- Website: https://www.eddiejaoude.io 
+- GitHub: https://github.com/eddiejaoude
+- YouTube: https://youtube.com/eddiejaoude
+- Twitter: https://twitter.com/eddiejaoude
+- Instagram: https://instagram.com/eddiejaoude
+- LinkedIn: https://linkedin.com/in/eddiejaoude
+- GitHub Community Org: https://github.com/EddieHubCommunity
+
+## Further
+Join Our Conversation At Discord: https://discord.eddiehub.org/
```

Comment:
Hey! The discord link isn't working, I have checked with the working link it contains **http** I wonder why it happens
have a look at it once [http://discord.eddiehub.org/](http://discord.eddiehub.org/)

Reply:
oh wupps slipped past me - http:///discord.eddiehub.org/ would be the correct link

Reply:
@Vyvy-vi may I know why it is not working for https 🤔

Reply:
> @Vyvy-vi may I know why it is not working for https 🤔 

Why?

Reply:
The discord link sub-domain is a URL-forward, instead of a normal CNAME or A level record(since discord's invite is case sensitive).
And thus, I think it doesn't get the SSL cert. from the root domain(if Eddie's provider supports that thing)
But, if Eddie's domain provider doesn't support inheriting SSL certs to sub-domains, then this just ends up as http.

There might be a way to make this work with https tho, with something like cloudfare acting as an intermediary

Reply:
It's a DNS level redirect, which doesn't support HTTPS. You can't use a CNAME or A record because those don't accept URL paths. 

It would be possible, however, to spin up a small web server to handle the redirect and maintain HTTPS.